apps/ios/SimpleXChat/FileUtils.swift

```diff
@@ -69,13 +69,29 @@ func fileModificationDate(_ path: String) -> Date? {
     }
 }
 
+public func deleteAppDatabaseAndFiles() {
+    let fm = FileManager.default
+    let dbPath = getAppDatabasePath().path
+    do {
+        try fm.removeItem(atPath: dbPath + CHAT_DB)
+        try fm.removeItem(atPath: dbPath + AGENT_DB)
```

Comment:
why are only these in try catch?

Reply:
others won't through too. Not sure why like this :)

Reply:
Because in case error happens here, would be useful to see it in logs. Bak files might not exist yet, so don't care about possible error of deleting them